### ISSUE TITLE
MODELS?

### ISSUE THREAD

**Issue by wsprogrammeryang** [machanic/SimulatorAttack#3]
hello，
i try to run
(2) Generate training data.
Generate training data of CIFAR-10 dataset by performing untargeted Bandits attack under L2 norm.
python of meta_simulator_bandits/learning/script/generate_bandits_training_data_script.py --gpu 0 --dataset CIFAR-10 --batch-size 100 --loss cw --norm l2
but   “/train_pytorch_model/real_image_model/CIFAR-10-pretrained/......*'” file doesn't exist.

then i go to the link you give but find no target model in the "simulator_attack / real_image_model / CIFAR-10-pretrained"file  compared to the MODELS_TRAIN_STANDARD in config.py

how to solve it?

**Comment by machanic** [comment]
Sorry for delay, I have uploaded all the models (including the models used in the training) to [Tsinghua Cloud Disk](https://cloud.tsinghua.edu.cn/d/a11beb12358b416199b9/).
You can download them.
